src/main/java/picard/illumina/parser/TileMetricsUtil.java

```diff
@@ -51,33 +54,76 @@
  * @author mccowan
  */
 public class TileMetricsUtil {
-    /** The path to the directory containing the tile metrics file relative to the basecalling directory. */
+
+    private static final Log log = Log.getInstance(TileMetricsUtil.class);
+
+    public static String CYCLE_TWENTY_FIVE_NAME = "C25.1";
```

Comment:
The name should reflect why it's important, e.g. TILE_METRICS_CYCLE_NAME. javadoc would be helpful too.